srcpkgs/python3-pyperclip/template

```diff
@@ -11,7 +11,11 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://pypi.org/project/pyperclip"
 distfiles="${PYPI_SITE}/p/pyperclip/pyperclip-${version}.tar.gz"
-checksum=9abef1e79ce635eb62309ecae02dfb5a3eb952fa7d6dce09c1aef063f81424d3
+checksum=105254a8b04934f0bc84e9c24eb360a591aaf6535c9def5f29d92af107a9bf57
+
+do_check() {
+	PYTHONPATH="$(cd build/lib* && pwd)" python3 tests/test_pyperclip.py
```

Comment:
why does the PYTHONPATH need to be evaluated? iirc the name of the lib dir should always be the same

Reply:
TBH I have no idea. I got that bit from when I read the python3-module build-style file for the first time and been using it since whenever I have to write a `do_check()`.

Reply:
I can't seem to make the skipped tests run locally even after adding `wl-clipboard` to the `checkdepends`. So probably have to do with `make_check=no` for now. But anyway this module uses python unittests so I don't think adding pytest works.

Reply:
[pytest can run unittest-based tests](https://docs.pytest.org/en/latest/how-to/unittest.html)

Reply:
Oh! totally missed that.